doc/BEGINNERS_TUTORIAL.org

```diff
@@ -0,0 +1,184 @@
+* Why Spacemacs?
+- Unparallelled text and structure editing for all types of writing tasks: creative writing, blogging, note-taking, todo-lists,
+  scientific papers... 
+- Powerful modes for programming in dozens of programming languages
+- Deeply customizable yet beginner-friendly
+* Installation
+ Spacemacs is a beginner-friendly and powerful distribution of a popular text
+ editor called Emacs. To install Spacemacs you need to first install base Emacs and then download
+ and install the Spacemacs extension files, which is easiest done by using a program
+ called Git. The steps are easy and outlined below.
+** 1. Install Emacs
+Follow the instructions in https://github.com/syl20bnr/spacemacs#prerequisites.
```

Comment:
I'm somewhat unfamiliar with this topic, but do you think this information is best placed in the beginners tutorial? If I understood correctly, the recommended build has this and http can be used, so by following these instructions, no problems should arise? I'm just thinking that the more concise the instructions are, the better, and maybe this information would find a better place [https://github.com/syl20bnr/spacemacs#prerequisites](here)?